? COPYING
? INSTALL
? depcomp
? gtk-doc.make
? install-sh
? missing
? mkinstalldirs
? demos/.deps
? demos/.libs
? libfoocanvas/.deps
? libfoocanvas/.libs
? libfoocanvas/foo-canvas-line.lo
? libfoocanvas/foo-canvas-marshal.lo
? libfoocanvas/foo-canvas-pixbuf.lo
? libfoocanvas/foo-canvas-polygon.lo
? libfoocanvas/foo-canvas-rect-ellipse.lo
? libfoocanvas/foo-canvas-text.lo
? libfoocanvas/foo-canvas-util.lo
? libfoocanvas/foo-canvas-widget.lo
? libfoocanvas/foo-canvas.lo
? libfoocanvas/libfoocanvas.la
? libfoocanvas/libfoocanvastypes.lo
? po/Makefile.in.in
Index: autogen.sh
===================================================================
RCS file: /cvs/gnome/foocanvas/autogen.sh,v
retrieving revision 1.2
diff -u -r1.2 autogen.sh
--- autogen.sh	21 Feb 2005 12:18:37 -0000	1.2
+++ autogen.sh	26 Mar 2006 13:12:40 -0000
@@ -41,5 +41,5 @@
   exit 1
 fi
 
-REQUIRED_AUTOMAKE_VERSION=1.5
+REQUIRED_AUTOMAKE_VERSION=1.9
 GNOME_DATADIR="$gnome_datadir" USE_GNOME2_MACROS=1 . $gnome_autogen
Index: configure.in
===================================================================
RCS file: /cvs/gnome/foocanvas/configure.in,v
retrieving revision 1.7
diff -u -r1.7 configure.in
--- configure.in	18 Sep 2005 20:18:11 -0000	1.7
+++ configure.in	26 Mar 2006 13:12:41 -0000
@@ -3,7 +3,7 @@
 AC_CONFIG_SRCDIR([libfoocanvas/foo-canvas.c])
 
 AC_CONFIG_HEADERS([config.h])
-AM_INIT_AUTOMAKE([dist-bz2])
+AM_INIT_AUTOMAKE([])
 
 AM_MAINTAINER_MODE
 
